controllers/apply/form-router-next/steps.js

```diff
@@ -172,6 +172,23 @@ module.exports = function(formId, formBuilder) {
                 stepFields.map(f => f.name).includes(item.param)
             );
 
+            function prevOrNextBtnNotClicked() {
```

Comment:
Not sure about this name? Sounds like a double negative with `not` in the name, can name this what it does. i.e. shouldRenderErrors?

Reply:
I didnt want to include the `errors` into this check tbh as I wanted to use them separately. I could combine them and named as `shouldRenderErrors`.